### DISCUSSION TITLE
API calls to export grading data

### DISCUSSION THREAD

**Discussion by saraswathi-pasumarthi** [jupyter/nbgrader#1673] [General]
I have added python files in nbgrader folder but are not getting reflected when deployed through kubernetes. Can anyone help me regarding this.

**Comment by perllaghu** [comment]
I assume that you have a docker image, with a notebook server & nbgrader installed?

If you start the image, and open a terminal... are the files present in whichever directory you copied them to?

If the files should be executable scripts, is that folder in `$PATH`?

If the files should be libraries, https://stackoverflow.com/questions/135035/python-library-path will help you check if the folder in in the library path
